tests/util/golinter/linter/whitelist.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package linter
+
+const (
+	// SkipAllRules could be used to skip all rules for a file path in WhitelistPath.
+	SkipAllRules = "SkipAllRules"
+)
+
+// WhitelistPath contains pairs of file and rule IDs. Each pair specifies which rules should not apply
+// to the file path in that pair. rule IDs are separated by comma.
+// Rule ID is the file name of each rule without ".go" extension.
+var WhitelistPath = map[string]string{}
```

Comment:
I think the map value needs to be a list or array, as we can whitelist multiple rules for one path.

Reply:
Done. Thanks.